WordSearchTest.java

```diff
@@ -0,0 +1,22 @@
+import org.junit.Test;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+public class WordSearchTest {
+    @Test
+    public void testFindAllWords() {
+        List<String> myDictionary = Arrays.asList("CAR", "CARD", "CART", "CAT");
+        List<String> prefixes = Arrays.asList("C", "CA", "CAR", "CARD", "CART", "CAT");
+        Dictionary dictionary = new Dictionary(myDictionary, prefixes);
+
+        WordSearch wordSearch = new WordSearch();

```

Comment:
Usual practice is to create tested object in a method annotated with `@Before` and just using it in test methods.